api/apps/search/queries.coffee

```diff
@@ -0,0 +1,19 @@
+module.exports =
+  matchAll: (term) ->
+    {
+      bool: {
+        must: {
+          multi_match: {
+            query: term,
+            type: 'best_fields',
+            fields: ['name.*', 'alternate_names.*'],
+            fuzziness: 2
+          }
+        }
+        should: {
+          match_phrase: {
+            name: term
+          }
+        }
+      }
+    }
```

Comment:
:+1: nice thing about json queries is you can just copy them between client apps.